### PR TITLE
Remove steps to build/test carma_msgs repo from CI.

### PR DESCRIPTION
**Pull request by paulbourelly999**
# PR Details
## Description
This PR is to remove steps to build and run unit test for the carma-msgs repo. This already happens in carma-msgs repo CI and is not necessary for generating sonar scanner report. **Update**: Removed entire circle ci workflow since project does not require unit test code coverage it can use [automatic analysis](https://docs.sonarsource.com/sonarcloud/advanced-setup/automatic-analysis/?_gl=1*bpmykr*_gcl_aw*R0NMLjE3MDU0MjI3MDAuQ2p3S0NBaUE3NWl0QmhBNkVpd0FraG85ZTkyNGFDb2JVbjNKOHJJcWdzLUNZd2xsU1hVbGJaSnFlUFZybmYtZ0F0Z2hoRmNFcnVrQUZSb0NJOEVRQXZEX0J3RQ..*_gcl_au*MjAwNDcxNzU4Ny4xNzA5NzYyNjAz*_ga*MTc2NDc2NjYwNC4xNjc4MjAyNTQx*_ga_9JZ0GZ5TC6*MTcxMjkyODQ3Mi43MC4xLjE3MTI5Mjk1NDcuNjAuMC4w) from sonar scanner. This does not require a CI process and instead just runs static analysis on every PR and on every push to default branch

## Types of changes

- [ ] Defect fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING**](https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/Contributing.md) document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by MishkaMN — APPROVED** [review]
if we are removing this CI file, this report will automatically be gone from the next PR right?
image

If there is no other configuration place to turn it off, this looks good to me.

File: .sonarqube/sonar-scanner.properties [hunk 1]
```diff
@@ -14,14 +14,13 @@
...
-sonar.projectKey=usdot-fhwa-stol_CARMAWebUi
+sonar.projectKey=usdot-fhwa-stol_carma-web-ui
 sonar.organization=usdot-fhwa-stol
 sonar.cfamily.cache.enabled=false
 sonar.cfamily.compile-commands=/opt/carma/build/compile_commands.json
 sonar.host.url=https://sonarcloud.io
-sonar.sources=website/scripts/, website/utils
+sonar.sources=website/
 # Set Git as SCM sensor
 sonar.scm.disabled=false
```
Comment (dan-du-car) [inline .sonarqube/sonar-scanner.properties:24]: Can we remove this sonar.scm.enabled=true if disabled is false?

**Comment by paulbourelly999** [comment]
> if we are removing this CI file, this report will automatically be gone from the next PR right? image
> 
> If there is no other configuration place to turn it off, this looks good to me.

Removed

File: .sonarqube/sonar-scanner.properties (hunk 1 above)
Comment (paulbourelly999) [reply .sonarqube/sonar-scanner.properties]: Done

**Pull request merged**